package.json

```diff
@@ -6,6 +6,9 @@
     "type": "git",
     "url": "https://github.com/fabric-design/styleguide"
   },
+  "scripts": {
+    "au": "au"
```

Comment:
whats this for?

Reply:
It allows to usejust the local installed./node_modules/.bin/au cli instead of installing it globally.

I just thought this reduces the need for global scripts.